releasenotes-builder/src/test/java/com/github/checkstyle/MainTest.java

```diff
@@ -0,0 +1,217 @@
+
+        // TwitterPublisher static
+        mockStatic(TwitterPublisher.class);
+    }
+
+    @Before
+    public void reset() {
+        TEST_COMMITS.clear();
+    }
+
+    @Test
+    public void testNoCommits() throws Exception {
+        assertMainReturnCode(0,
+                "-localRepoPath", temporaryFolder.newFolder().getAbsolutePath(),
+                "-remoteRepoPath", "checkstyle/checkstyle",
+                "-startRef", "12345678",
+                "-releaseNumber", "10.0.1",
+                "-outputLocation", temporaryFolder.newFolder().getAbsolutePath(),
+                "-githubAuthToken", "TOKEN",
+                "-generateAll",
+                "-publishTwit",
+                "-twitterConsumerKey", "KEY",
+                "-twitterConsumerSecret", "SECRET",
+                "-twitterAccessToken", "TOKEN",
+                "-twitterAccessTokenSecret", "SECRET",
+                "-validateVersion"
+        );
+    }
+
+    @Test
+    public void testUnknownCommit() throws Exception {
+        TEST_COMMITS.add(RevCommitUtil.create("Hello World"));
+
+        assertMainReturnCode(0,
+                "-localRepoPath", temporaryFolder.newFolder().getAbsolutePath(),
+                "-remoteRepoPath", "checkstyle/checkstyle",
+                "-startRef", "12345678",
+                "-releaseNumber", "10.0.1",
+                "-outputLocation", temporaryFolder.newFolder().getAbsolutePath(),
+                "-githubAuthToken", "TOKEN",
+                "-generateAll",
+                "-publishTwit",
+                "-twitterConsumerKey", "KEY",
+                "-twitterConsumerSecret", "SECRET",
+                "-twitterAccessToken", "TOKEN",
+                "-twitterAccessTokenSecret", "SECRET",
+                "-validateVersion"
+        );
+    }
+
+    @Test
+    public void testIssueCommitWithIssueNotFound() throws IOException {
+        TEST_COMMITS.add(RevCommitUtil.create("Issue #1: Hello World"));
+
+        assertMainReturnCode(-2,
+                "-localRepoPath", temporaryFolder.newFolder().getAbsolutePath(),
+                "-remoteRepoPath", "checkstyle/checkstyle",
+                "-startRef", "12345678",
+                "-releaseNumber", "10.0.1",
+                "-outputLocation", temporaryFolder.newFolder().getAbsolutePath(),
+                "-githubAuthToken", "TOKEN",
+                "-generateAll",
+                "-publishTwit",
+                "-twitterConsumerKey", "KEY",
+                "-twitterConsumerSecret", "SECRET",
+                "-twitterAccessToken", "TOKEN",
+                "-twitterAccessTokenSecret", "SECRET",
+                "-validateVersion"
+        );
+    }
+
+    @Test
+    public void testIssueCommit() throws Exception {
+        TEST_COMMITS.add(RevCommitUtil.create("Issue #1: Hello World"));
+        TEST_ISSUES.add(GhIssueUtil.create(1, GHIssueState.OPEN, "Hello World",
+                Constants.MISCELLANEOUS_LABEL));

```

Comment:
`TEST_ISSUES.add(GhIssueMock.of(1, OPEN, "Hello World",MISC));`

Can we archive this ? By renaming, static import and some synonym local constants.

Reply:
> static import 

Done for GHIssueState and Constants.

> Can we archive this 

Not following what is meant by "Archive".

> GhIssueMock.of

So I should rename `RevCommitUtil.create` and `GhIssueUtil.create` to follow the pattern of `*Mock.of`?